src/app/view/components/login/login-component/login.component.ts

```diff
@@ -111,6 +111,7 @@ class LoginComponent {
                 error += this.mandatoryFieldError.replace('{0}', this.passwordLabel);
             }
             this.message = error;
+            this.dotLoadingIndicatorService.hide();
```

Comment:
The same before Why do you need it right here?, you are showing or hiding the loading indicator in LoginContainer so I think you dont need it in this Component